uploader.json

```diff
@@ -1372,6 +1372,11 @@
             "pattern": [],
             "symbolic_name": "ifcfg"
         },
+        {
+            "file": "/etc/sysconfig/network-scripts/()*route-.*",
+            "pattern": [],
+            "symbolic_name": "route"
```

Comment:
The symbolic_name needs to match the spec name used in insights-core, please change this to `ifcfg_static_route`.